app/assets/javascripts/discourse/app/components/user-menu/notifications-list.js

```diff
@@ -70,8 +71,10 @@ export default class UserMenuNotificationsList extends UserMenuItemsList {
     return this.store
       .findStale("notification", params)
       .refresh()
-      .then((c) => {
-        return c.content.map((notification) => {
+      .then(async (c) => {
+        const notifications = c.content;
+        await Notification.applyTransformations(notifications);
+        return notifications.map((notification) => {
```

Comment:
Feels a little strange to be combining `.then` syntax and `async/await`. Can we do something like

```javascript
const refreshedNotifications = await this.store.findStale("notification", params).refresh()
await Notification.applyTransformations(refreshedNotifications);
return notifications.map((notifications) => {
...
```